mono/tests/bug-8417.cs

```diff
@@ -0,0 +1,93 @@
+#define USE_REDIRECT
```

Comment:
This test should probably be added to https://github.com/mono/mono/blob/master/mcs/class/System/Test/System.Diagnostics/ProcessTest.cs instead